linux_os/guide/services/ssh/ssh_client/ssh_client_use_approved_ciphers_ordered_stig/oval/shared.xml

```diff
@@ -0,0 +1,49 @@
+{{%- if product == 'ubuntu2404' %}}
+{{%- set ssh_approved_ciphers="aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes128-ctr" %}}
+{{%- endif %}}
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("Limit the ciphers to those which are FIPS-approved.", rule_title=rule_title) }}}
+    <criteria operator="AND">
+      <extend_definition comment="Installed OS is FIPS certified" definition_ref="installed_OS_is_FIPS_certified" />
```

Comment:
I'd suggest stepping down from this extend_definition. We use it at multiple places but I think we shouldn't use it. The reason is that operating systems aren't FIPS-certified, instead, specific cryptography modules are FIPS-certified.